packages/ui/Scrollable/Scrollable.js

```diff
@@ -64,710 +64,499 @@ const TouchableDiv = ForwardRef({prop: 'ref'}, Touchable('div'));
+
+		// constants
+		pixelPerLine: 39,
+		scrollWheelMultiplierForDeltaPixel: 1.5, // The ratio of wheel 'delta' units to pixels scrolled.
+
+		// status
+		deferScrollTo: true,
+		isScrollAnimationTargetAccumulated: false,
+		isUpdatedScrollThumb: false,
+
+		// overscroll
+		lastInputType: null,
+		overscrollStatus: {
+			horizontal: {
+				before: {type: overscrollTypeNone, ratio: 0},
+				after: {type: overscrollTypeNone, ratio: 0}
+			},
+			vertical: {
+				before: {type: overscrollTypeNone, ratio: 0},
+				after: {type: overscrollTypeNone, ratio: 0}
+			}
+		},
+
+		// bounds info
+		bounds: {
+			clientWidth: 0,
+			clientHeight: 0,
+			scrollWidth: 0,
+			scrollHeight: 0,
+			maxTop: 0,
+			maxLeft: 0
+		},
 
-		this.resizeRegistry = Registry.create(this.handleResize.bind(this));
+		// wheel/drag/flick info
+		wheelDirection: 0,
+		isDragging: false,
+
+		// scroll info
+		scrolling: false,
+		scrollLeft: 0,
+		scrollTop: 0,
+		scrollToInfo: null,
+
+		// component info
+		childRefCurrent: null,
+
+		// scroll animator
+		animator: null,
+
+		// non-declared-variable.
+		accumulatedTargetX: null,
+		accumulatedTargetY: null,
+		flickTarget: null,
+		dragStartX: null,
+		dragStartY: null,
+		scrollStopJob: null,
+
+		isScrollbarVisibleChanged: false
+	});
+
+	useImperativeHandle(reference, () => ({
+		animator: variables.current.animator,
+		applyOverscrollEffect,
+		bounds: variables.current.bounds,
+		canScrollHorizontally,
+		canScrollVertically,
+		checkAndApplyOverscrollEffect,
+		get childRefCurrent () {
+			return variables.current.childRefCurrent;
+		},
+		containerRef,
+		getScrollBounds,
+		horizontalScrollbarRef,
+		get isDragging () {
+			return variables.current.isDragging;
+		},
+		isScrollAnimationTargetAccumulated: variables.current.isScrollAnimationTargetAccumulated,
+		get lastInputType () {
+			return variables.current.lastInputType;
+		},
+		set lastInputType (val) {
+			variables.current.lastInputType = val;
+		},
+		props: {
+			rtl: props.rtl
+		},
+		get scrollLeft () {
+			return variables.current.scrollLeft;
+		},
+		scrollTo,
+		scrollToAccumulatedTarget,
+		get scrollToInfo () {
+			return variables.current.scrollToInfo;
+		},
+		get scrollTop () {
+			return variables.current.scrollTop;
+		},
+		setOverscrollStatus,
+		showThumb,
+		start,
+		startHidingThumb,
+		verticalScrollbarRef,
+		get wheelDirection () {
+			return variables.current.wheelDirection;
+		},
+		set wheelDirection (val) {
+			variables.current.wheelDirection = val;
+		}
+	}));
+
+	// TODO: consider replacing forceUpdate() by storing bounds in state rather than a non-
+	// state member.
+	const enqueueForceUpdate = useCallback(() => {
+		variables.current.childRefCurrent.calculateMetrics(variables.current.childRefCurrent.props);
+		forceUpdate();
+	}, []);
+
+	const handleResizeWindow = useCallback(() => {
+		// `handleSize` in `ui/resolution.ResolutionDecorator` should be executed first.
+		setTimeout(() => {
+			if (handleResizeWindow) {
```

Comment:
In the original code, there are two `handleResizeWindow` functions. One is from props and the other one is a member function of a component. We need to check the existence and call `props.handleResizeWindow` here. (We need to put it into dependency array, too)

Reply:
It was my mistake. I fixed it. Thank you.